core/src/test/scala/org/apache/spark/executor/ExecutorSuite.scala

```diff
@@ -270,29 +270,36 @@ class ExecutorSuite extends SparkFunSuite
     heartbeatZeroAccumulatorUpdateTest(false)
   }
 
+  private def withMockHeartbeatReceiverRef(executor: Executor)
+                                          (func: RpcEndpointRef => Unit): Unit = {
+    val executorClass = classOf[Executor]
+    val mockReceiverRef = mock[RpcEndpointRef]
+    val receiverRef = executorClass.getDeclaredField("heartbeatReceiverRef")
+    receiverRef.setAccessible(true)
+    receiverRef.set(executor, mockReceiverRef)
+
+    func(mockReceiverRef)
+  }
+
   private def withHeartbeatExecutor(confs: (String, String)*)
-      (f: (Executor, ArrayBuffer[Heartbeat]) => Unit): Unit = {
+                                   (f: (Executor, ArrayBuffer[Heartbeat]) => Unit): Unit = {
```

Comment:
Please avoid re-indenting existing code!

Here the old indention is the correct one so please follow that in line 274 too.
You can find the details about indentation [here](https://github.com/databricks/scala-style-guide#spacing-and-indentation).

Reply:
Done.